app/helpers/provider/admin/dashboards_helper.rb

```diff
@@ -19,17 +19,22 @@ def sign_class(widget)
     widget.percentual_change > 0 ? 'u-plus' : 'u-minus'
   end
 
-  def dashboard_collection_link(singular_name, collection, path, plural = nil, icon_name = nil)
+  def dashboard_collection_link(singular_name, path, opts)
     link_to path, class: 'DashboardNavigation-link' do
-      link_text = pluralize(number_to_human(collection.size), singular_name, plural)
-      link_text = link_text.prepend "#{icon(icon_name)} " if icon_name
+      if opts[:collection]
+        link_text = pluralize(number_to_human(opts[:collection].size), singular_name, opts[:plural])
```

Comment:
The idea of pluralise is that you give it the singular and it gives you the plural back. Not sure I follow this change.

Reply:
The purpose of the method wasn't clear but I realize now it is precisely to pluralize a link, isn't it? So I should not have made `collection` an optional argument.